app/controllers/charges_controller.rb

```diff
@@ -81,13 +81,13 @@ def refund
     end
 
     req_id = @charge['metadata']['requirement_id']
-    reg_id = @charge['metadata']['registration_id']
+    # this supports both the old registration_id and the newer team_id
+    team_id = @charge['metadata']['team_id'] || @charge['metadata']['registration_id']
 
-    cr = CompletedRequirement.where(:requirement_id => req_id, :registration_id => reg_id).first
-    reg = cr.registration
+    cr = CompletedRequirement.where(:requirement_id => req_id, :team_id => team_id).first
```

Comment:
TODO: store something other than a team_id and requirement_id in the stripe metadata to account for situations where db ids may change (like migrations)